client/src/app/components/edit-profile/edit-profile.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="md-dialog.fullscreen-dialog">
+    <div class="wrapper">
+        <div class="cover-picture">
+            <img src="https://ollienollie.files.wordpress.com/2009/10/untitled-181.jpg" class="profile-pic" />
+            <button class="dot">
+                <mat-icon id="add-a-photo" aria-hidden="false">add_a_photo</mat-icon>
+            </button>
+            <button mat-button class="edit-cover-picture">
+                <mat-icon id="add-a-photo" aria-hidden="false">add_a_photo</mat-icon> <span class="edit-cover-pic-text">Edit Cover Photo</span>
+            </button>
+        </div>
+
+        <div class="container">
+            <form>
```

Comment:
Question: Is the div with the class container needed if you can add that class to the form?